### PR TITLE
add community card component

### LINKED ISSUE #35: Community Page - Community Card
Client only issue.
Make the card that is displayed in the community page.
Use fake data for the street and people for now.
Make room for the map to the right but you can use a placeholder image instead of the map and marker.

image

### PR DESCRIPTION
**Pull request by RedPenguin88**
This closes #35.
### Motivation and Context
For the community page, there needs to be a way to cleanly look at a community showing basic information including community name, location, the people in said community, and more.

This PR serves as a basic template for such a page, as the card contains dummy data for now. The component can be imported:
```ts
import { CommunityCard } from '@/components';
```
and then included by:
```ts
<CommunityCard />
```

Here is an image of what it currently looks like:
community_page
### Types of Changes
- [x] Utilizes MUI's card component which can easily be extended in future PRs
- [x] Currently uses dummy data for the location and personnel
- [x] Currently uses a temporary image (Roadwatch Logo). For the finished component, this would display the location of the community. See #35 for the reference image
### Potential Future Updates
- [ ] Replace the fake data for location and personnel with real data
- [ ] The temp image should show a map which shows the location of the community

### CODE REVIEW

**Review by Linkaloo — APPROVED** [review]
looks very nice

File: client/src/components/CommunityCard/CommunityCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.CommunityCard {
+
```
Comment (Aligary) [inline client/src/components/CommunityCard/CommunityCard.module.scss:2]: We should add the styling in the .scss file

**Review by Aligary — APPROVED** [review]
My comments aren't blockers for this PR so I am going to approve it. Looks good!

**Review by morenj — APPROVED** [review]
I tested and reviewed the code, and everything looks good. I agree with Aligary regarding moving the CSS to the .scss file for consistency across the project. 

Great work!

We can however merge as is for now, since we want to keep things moving along and this maybe something we can improve in the future if time allows.

**Review by trevortippery — APPROVED** [review]
Looks great. The only thing that could change is having a more descriptive alt prop on your  tag. It could be "Displaying location for the Community Page."

Here is a link about this topic: https://wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/

**Pull request merged**